src/js/view/components/nextuptooltip.js

```diff
@@ -139,8 +139,8 @@ define([
                 this.showNextUp = false;
                 return;
             }
-
-            this.showNextUp = true;
+            var showNextUp = nextUp.showNextUp;
+            this.showNextUp = (showNextUp === undefined) ? true : showNextUp;
```

Comment:
This isn't intuitive. I think it's better to  set `showNextUp` for playlist mode in the other PR.